prost-build/Cargo.toml

```
@@ -25,7 +25,7 @@ bytes = { version = "1", default-features = false }
 heck = "0.4"
 itertools = { version = "0.10", default-features = false, features = ["use_alloc"] }
 log = "0.4"
-multimap = { version = "0.8", default-features = false }
+multimap = { version = ">=0.8, <0.10", default-features = false }
```

why less than 0.10?

---

Personally, I think this syntax explains better the intent of the version bounds.
```suggestion
multimap = { version = ">=0.8, <=0.9", default-features = false }
```